packages/components/tag/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@contentful/f36-tag",
+  "version": "4.0.0",
```

Comment:
Will this be available instantly as version `4.0.0`? maybe `alpha`?

Reply:
once merged to `forma-v4` it will released as `alpha`, according to CI setup